app/src/main/java/com/bnyro/recorder/ui/models/TrimmerModel.kt

```diff
@@ -25,17 +26,16 @@ class TrimmerModel(context: Context) : ViewModel() {
     var startTimeStamp by mutableLongStateOf(0L)
     var endTimeStamp by mutableStateOf<Long?>(null)
 
+    @RequiresApi(Build.VERSION_CODES.O)
     fun startTrimmer(context: Context, inputFile: DocumentFile) {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            viewModelScope.launch {
-                val trimmer = MediaTrimmer()
-                Toast.makeText(context, "Starting Trimmer", Toast.LENGTH_LONG).show()
-                val result = trimmer.trimMedia(context, inputFile, startTimeStamp, endTimeStamp!!)
-                if (result) {
-                    Toast.makeText(context, "Trim Success", Toast.LENGTH_LONG).show()
-                } else {
-                    Toast.makeText(context, "Trim Failed", Toast.LENGTH_LONG).show()
-                }
+        viewModelScope.launch {
+            val trimmer = MediaTrimmer()
+            Toast.makeText(context, "Starting Trimmer", Toast.LENGTH_LONG).show()
+            val result = trimmer.trimMedia(context, inputFile, startTimeStamp, endTimeStamp!!)
+            if (result) {
+                Toast.makeText(context, "Trim Success", Toast.LENGTH_LONG).show()
+            } else {
+                Toast.makeText(context, "Trim Failed", Toast.LENGTH_LONG).show()
```

Comment:
Please make all these strings above translatable too.

Reply:
I feel like showing just a Toast is not enough. Do you agree?

Reply:
This is the thing, Trimming can take some time to complete. So showing "Trimming started" is no problem.
However, if I start trim and leave the phone for a while, there's no way to know if the trim is success or failed.

Also trimming is running inside a viewModel scope. So we can't let the user navigate away from the trimmer screen while the trim is ongoing.

Reply:
Hmm, that makes sense, I agree. 
I think it would be the best to use `CoroutineScope(Dispatchers.IO)` then and show a success/failure notification.